packages/table/src/cell/editableCell.tsx

```diff
@@ -57,6 +57,7 @@ export class EditableCell extends React.Component<IEditableCellProps, {}> {
                         className={"bp-table-editable-name"}
                         defaultValue={value}
                         intent={intent}
+                        minWidth={null}
```

Comment:
shouldn't we limit it to _something_, like 5px maybe? also, minor: it always feels weird to pass `null` as a prop.